src/node_crypto.h

```diff
@@ -59,10 +60,51 @@
 # define NODE__HAVE_TLSEXT_STATUS_CB
 #endif  // !defined(OPENSSL_NO_TLSEXT) && defined(SSL_CTX_set_tlsext_status_cb)
 
+#define DEFAULT_CIPHER_LIST_V10_38 "ECDHE-RSA-AES128-SHA256:"                  \
+                                "AES128-GCM-SHA256:RC4:HIGH:!MD5:!aNULL:!EDH"
+
+#define DEFAULT_CIPHER_LIST_V10_39 "ECDHE-RSA-AES128-SHA256:"                  \
+                                "AES128-GCM-SHA256:HIGH:!RC4:!MD5:!aNULL:!EDH"
+
+#define DEFAULT_CIPHER_LIST_V12_2 "ECDHE-RSA-AES128-SHA256:"                   \
+                                "DHE-RSA-AES128-SHA256:AES128-GCM-SHA256:RC4:" \
+                                "HIGH:!MD5:!aNULL"
+
+#define DEFAULT_CIPHER_LIST_V12_3  "ECDHE-RSA-AES128-SHA256:"                  \
+                                "DHE-RSA-AES128-SHA256:AES128-GCM-SHA256:HIGH:"\
+                                "!RC4:!MD5:!aNULL"
+
+#define DEFAULT_CIPHER_LIST_HEAD "ECDHE-RSA-AES256-SHA384:"                   \
+                                 "DHE-RSA-AES256-SHA384:"                     \
+                                 "ECDHE-RSA-AES256-SHA256:"                   \
+                                 "DHE-RSA-AES256-SHA256:"                     \
+                                 "ECDHE-RSA-AES128-SHA256:"                   \
+                                 "DHE-RSA-AES128-SHA256:"                     \
+                                 "HIGH:!aNULL:!eNULL:!EXPORT:!DES:!RC4:!MD5:" \
+                                 "!PSK:!SRP:!CAMELLIA"
```

Comment:
alignment issues on the ending `/`. Check that all the `DEFAULT_CIPHER_LIST_*` are aligned at the end the same.